### DISCUSSION TITLE
How to use php with fast-cgi webserver example

### DISCUSSION THREAD

**Discussion by param-esec** [AxisCommunications/acap-native-sdk-examples#163] [Q&A] [answered]
I would like to create a webpage using php to retrieve user's input and settings. I want to confirm if php is supported by ACAP. If not, could you please suggest an alternative method to obtain user inputs? Can I achieve this using html and javascript?

**Comment by pataxis** [comment, marked as answer]
Hi @param-esec , thanks for your question!

We don't have any example today but javascript is often used in ACAP applications.

Here's a small outline how I would go about to try this:
* Based on the [web-server-using-fastcgi](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/web-server-using-fastcgi) example.
* Create a directory **html** in ACAP application project, i.e. under **app** directory.
* Create a HTML page **index.html** in the **html** directory.
* Add to **manifest.json** field  `acapPackageConfconfiguration.settingPage` value `index.html`.
* Add javascript functions to **index.html** for e.g. fields and buttons and try to send user input to the CGI.
